packages/idempotency/tests/unit/makeFunctionIdempotent.test.ts

```diff
@@ -25,6 +27,12 @@ const mockGetRecord = jest
   .spyOn(BasePersistenceLayer.prototype, 'getRecord')
   .mockImplementation();
 
+const mockLambaContext: Context = {
```

Comment:
Nit: consider importing the dummy context [from here](https://github.com/aws-powertools/powertools-lambda-typescript/blob/main/packages/commons/src/samples/resources/contexts/hello-world.ts) like we do [in other tests](https://github.com/aws-powertools/powertools-lambda-typescript/blob/main/packages/metrics/tests/unit/Metrics.test.ts#L6-L10).

Reply:
ah, I was looking for something similar. We need to move this thing to a better place, somewhere else.